terraform/modules/rds-serverless/variables.tf

```diff
@@ -0,0 +1,75 @@
+
```

Comment:
most of those variables are nice but unnecessary, we can simplify them by hardcoding their values because this module is for organisation purposes only (we are not going to reuse them, we will create only one database from it)

if at some point we decide that they need to be customised then we can convert to a variable (YAGNI principle: you ain't gonna needed)